dashboard/app/models/script.rb

```diff
@@ -122,7 +122,7 @@ def generate_plc_objects
   end
 
   serialized_attrs %w(
-    hideable_stages
+    hideable_lessons
```

Comment:
Have you thought through whether this will cause any problems during deploy? Please poke a hole in my sequencing:
1. production-daemon runs `rake seed`, which populates the DB from the .script files
2. production database now contains Script objects with `hideable_lessons` but not `hideable_stages`
3. existing production servers read the `hideable_lessons` field on various scripts using the script cache. no problem, because the old script objects are still cached
4. do existing production servers query the DB for script objects, without using the cache? if they do, this data won't be in a format it can understand, because it will still be looking for hideable_stages, but only hideable_lessons will be there.

I don't know for sure whether step 4 actually happens or not. possible paths forward:
A. comb through this PR for any examples and argue that it does not contain any examples of step 4 above
B. keep both properties on the script object for a deploy or two, like we've been doing for the js caching issue. (the server code does not then need to check both values -- the old value is only for use by the code that's already running in production prior to this change shipping).
C. argue that the downtime * severity is low enough that we don't mind a short window of things not working properly